docs/security.html

```diff
@@ -1561,16 +1524,6 @@ <h4 class="anchor-heading"><a id="security_authz_examples" class="anchor-link"><
             <pre><code class="language-bash">$ bin/kafka-acls.sh --bootstrap-server localhost:9092 --add --allow-principal User:Bob --consumer --topic Test-topic --group Group-1 </code></pre>
             Note that for consumer option we must also specify the consumer group.
             In order to remove a principal from producer or consumer role we just need to pass --remove option. </li>
-
-        <li><b>Admin API based acl management</b><br>
```

Comment:
All the commands above already use `bootstrap-server` so this section is not needed anymore.